components/brave_rewards/resources/page/ping_doc_signer/components/Header/Header.tsx

```diff
@@ -92,13 +98,21 @@ export const Header: React.FC<HeaderProps> = ({
                 <StyledFadeAway fadeAnimation={isStatusVisible}>
                     <StyledHeaderButton pdfFile={pdfFile} onClick={handleSignButtonClick}>Add signature</StyledHeaderButton>
                     <StyledHeaderControlsBar />
-                    <StyledHeaderButton
-                        pdfFile={pdfFile}
-                        onClick={handleVerifyButtonClick}
-                        as={isVerified ? StyledVerified : isVerificationFailed ? StyledNotVerified : 'button'}
+                    <Tooltip
+                        text="Verification failed. Please check the document."
```

Comment:
we are not showing any new information here. the tooltip should convey extra information about verification failure like there are no signatures on this document or the attached signature is not valid, etc.